zulipterminal/helper.py

```diff
@@ -737,7 +737,9 @@ def suppress_output() -> Iterator[None]:
 
 
 @asynch
-def process_media(controller: Any, media_link: str) -> None:
+def process_media(
+    controller: Any, media_link: str, callback: Callable[..., None]
```

Comment:
callback is a very general name - it's worth clarifying when it's called in the name. This could be after downloading, during downloading...